web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -1334,6 +1334,15 @@ class LiftRules() extends Factory with FormVendor with LazyLoggable {
     }
   }
 
+  /**
+    * The root path for assets folder. For example to store
+    * asset files under 'myassets' folder, use value '/myassets/'. Assets
+    * under this configurable path follow conventions, for example
+    * the default lazy loading spinner ('ajax-loader.gif') is under
+    * 'images' folder.
+    */
+  @volatile var assetPath: String = "/"
```

Comment:
Would it make sense to have this default to `""` instead of removing the `/` from the links?

Reply:
@eltimn I was thinking that option too, but eventually leaned towards this way. I was thinking that slash would make it clear that it is about root path, and also when with a folder value it would need to be there. 

But, I'm happy with either way, just let me know. :)

Reply:
I'm fine either way, really. I just wanted to bring it up and see if anyone else had an opinion.

Reply:
There is also a third possible approach, I wrote to the mailing list about it:
https://groups.google.com/d/msg/liftweb/bf07dlUfWpc/cUG6KXeiAQAJ

Reply:
I'm down for the current approach. One thought though: should this be `assetRoot` or `assetPathRoot`?

Also, the scaladocs don't make it immediately clear that this is something that is relative to the domain root; that is, that this goes directly into HTML references and therefore `/` means something other than “project root” or any number of other possibilities. It's also worth noting that this will in turn be rewritten to take context path into account, if that is set, for maximum fun.

Reply:
Yes, maybe it would be better to make those things more clear, the naming and also scaladoc. I'll try to find time to change them today after work.